### PR TITLE
Bugfix/folder structure

### PR DESCRIPTION
**Pull request by sattyframework**
re-structured button styles so each styles is its own variant 
created the block button as a partial and pass in the variant classes

### CODE REVIEW

**Review by paulweareframework — APPROVED** [review]

File: resources/css/components/btn/btn-arrow.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@layer components {
+  .btn-arrow {
+    @apply inline-flex justify-center p-20 default-transition;
+  }
+
+  .btn-arrow--mustard {
+    @apply text-slate-dark bg-mustard disabled:bg-stone disabled:text-paleMint;
```
Comment (Dunsters99) [inline resources/css/components/btn/btn-arrow.css:7]: is there a hover state for this button?

File: resources/css/components/btn/btn-arrow.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+@layer components {
+  .btn-arrow {
+    @apply inline-flex justify-center p-20 default-transition;
```
Comment (Dunsters99) [inline resources/css/components/btn/btn-arrow.css:3]: if you want to centre elements inside, I'd add `items-center` as well

File: resources/css/components/btn/btn-block.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
+@layer components {
+  .btn-block {
+    @apply relative inline-flex justify-center uppercase font-neurial text-[12px] tracking-[0.12em] py-20 px-25  default-transition;
```
Comment (Dunsters99) [inline resources/css/components/btn/btn-block.css:3]: extra space at the end of the class list

File: resources/css/components/btn/btn-block.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+    &:hover:enabled,
+    &:focus {
+      .btn-block-bg--white {
+        @apply scale-110;
+      }
+    }
+  }
+
+  .btn-block-bg--garden {
```
Comment (Dunsters99) [inline resources/css/components/btn/btn-block.css:28]: I think the class naming structure here would be `.btn-block--garden` then the element `.btn-block__bg` would get the classes

File: resources/css/components/btn/btn-block.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    @apply absolute inset-0 bg-garden default-transition;
+  }
+
+  .btn-block-bg--white {
```
Comment (Dunsters99) [inline resources/css/components/btn/btn-block.css:32]: same here

File: resources/views/home.antlers.html [hunk 6]
```diff
@@ -28,9 +28,9 @@ <h5 class="heading-xs">Heading Five
 </div>
 
 <div class="container mb-10">
-  <button class="btn btn--garden">Primary Button</button>
-  <button disabled class="btn btn--white">Primary Button</button>
+  {{partial:global/btn/block class="btn-block--garden" inner_class="btn-block-bg--garden" label="Primary Button"}}
+  {{partial:global/btn/block class="btn-block--white" inner_class="btn-block-bg--white" label="Secondary Button"}}
```
Comment (Dunsters99) [inline resources/views/home.antlers.html:32]: I'm not sure I understand how/why the `inner_class` prop is used

**Review by Dunsters99 — CHANGES_REQUESTED** [review]

File: resources/css/components/btn/btn-arrow.css (hunk 1 above)
Comment (sattyframework) [reply resources/css/components/btn/btn-arrow.css:7]: There is, it's an animation where the arrow goes right continuously. I left it for now as I didn't want to spend too much time on setting up.

File: resources/views/home.antlers.html (hunk 6 above)
Comment (sattyframework) [reply resources/views/home.antlers.html:32]: Removed

File: resources/css/components/btn/btn-arrow.css (hunk 1 above)
Comment (sattyframework) [reply resources/css/components/btn/btn-arrow.css:7]: Nevermind the last comment, I've attempted the animation

**Review by Dunsters99 — APPROVED** [review]

**Pull request merged**